Source/Turbo.swift

```diff
@@ -17,6 +17,12 @@ public class TurboConfig {
         VisitableViewController(url: url)
     }
 
+    /// The navigation controller used in `TurboNavigator` for the main and modal stacks.
+    /// Must be a `UINavigationController` or subclass.
+    public var defaultNavigationController: () -> UINavigationController = {
+        UINavigationController()
+    }
```

Comment:
Given the motivation here is to customize `UIKit` presentations, do you reckon there could be a need for different controllers for the main and modal stacks? Or is that just not really plausible 🤔.

Reply:
I thought about it but decided against it. The `UINavigationController` subclass could always check if it is being presented to apply specific logic to modals.

Reply:
Ah that's a great point 👍